stan-dev/stan
# Feature/2563 users guide html

#### Submission Checklist

- [n/a] Run unit tests: `./runTests.py src/test/unit`
- [n/a] Run cpplint: `make cpplint`

#### Summary

* Convert users guide from LaTeX/pdf to bookdown/HTML+pdf 
* Add 100 pages of intro to BDA with examples
* Change title from *Stan User's Guide* to *Bayesian Statistics using Stan* to reflect expanded scope

#### Intended Effect

* Provide HTML doc for searchability on the web
* Provide markdown-based doc for ease of updates
* Keep pdf form of doc

#### How to Verify

The build target 

```
> make doc
```

will make both versions, as well as the reference manual and function reference.  
It should be built during continuous integration as it is delegated to in `make manual`.

#### Side Effects

Future manual changes will need to go into bookdown, so anything in process will need to be updated.  

#### Documentation

Uh, yes.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
